templates/parts/event-list.php

```diff
@@ -12,29 +12,21 @@
 /** @var ?bool $show_start */
 /** @var ?bool $show_end */
 /** @var ?bool $show_excerpt */
-/** @var ?string $date_format */
-/** @var ?bool $relative_time */
 /** @var ?string[] $extra_classes */
 /** @var ?Attendee[] $current_user_attendee_per_event Associative array with event id as key, and Attendee as value. */
 
 $show_start                      = $show_start ?? false;
 $show_end                        = $show_end ?? false;
 $show_excerpt                    = $show_excerpt ?? true;
-$date_format                     = $date_format ?? '';
-$relative_time                   = $relative_time ?? true;
 $show_permanent_delete           = $show_permanent_delete ?? false;
 $extra_classes                   = isset( $extra_classes ) ? implode( $extra_classes, ' ' ) : '';
 $current_user_attendee_per_event = $current_user_attendee_per_event ?? array();
 
 /**
  * @param Event_Start_Date|Event_End_Date $time
  */
-$print_time = function ( $time ) use ( $date_format, $relative_time ): void {
-	if ( $relative_time ) {
-		$time->print_relative_time_html( $date_format );
-	} else {
-		$time->print_time_html( $date_format );
-	}
+$print_time = function ( $time ) : void {
```

Comment:
We could remove the `$print_time` function as the logic it contained has now been moved to the `Event_Date`.